modules/ROOT/pages/proxy-mule3-elements.adoc

```diff
@@ -1,19 +1,18 @@
-= Mule 3.x API Proxy elements
+= Filtering Internal Mule HTTP Headers in Mule 3.x API Proxy
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-== Automatically Filtering Internal Mule HTTP Headers
 
-To avoid internal Mule HTTP headers from being leaked to the backend implementation or from being included in the response, use one of the following two filters, as appropriate:
+To avoid internal Mule HTTP headers from being leaked to the backend implementation or from being included in the response, Anypoint Platform automatically implements the following:
```

Comment:
They are not implemented by Anypoint Platform. I would say instead something like: API proxies by default filter them using the following components:

Reply:
Done! Thanks!

Reply:
As I said in my previous comment: "They are not implemented by Anypoint Platform".

Reply:
I also would remove the use of the word "automatically" here. I would like to disallow the notion that somehow the "automatic" process can be disabled.